src/wallet.rs

```diff
@@ -302,13 +302,14 @@ impl Wallet {
     commit: Transaction,
     reveal: Transaction,
     output: batch::Output,
+    no_wait: bool,
   ) -> Result<batch::Output> {
     eprintln!("Waiting for rune commitment {} to mature…", commit.txid());
 
     self.save_etching(rune, &commit, &reveal, output.clone())?;
 
     loop {
-      if SHUTTING_DOWN.load(atomic::Ordering::Relaxed) {
+      if SHUTTING_DOWN.load(atomic::Ordering::Relaxed) || no_wait {
```

Comment:
why not instead have a separate message for `no_wait` instead of combining it with `suspending batch....` message, else branch saying "commitment submiited, reveal is pending" something along that line.